raiden/blockchain/events.py

```diff
@@ -316,7 +316,7 @@ def add_token_network_registry_listener(
         self,
         token_network_registry_proxy: TokenNetworkRegistry,
         contract_manager: ContractManager,
-        from_block: BlockSpecification = "latest",
+        from_block: BlockNumber,
```

Comment:
I changed all of the `StatelessFilter` APIs to only accept `BlockNumber`, the goal was to remove the possibility of using tags, so that race conditions are not possible in that piece of code.